.gitignore

```diff
@@ -8,3 +8,4 @@ breach_hashsets
 version.json
 loadtests/__pycache__
 loadtests/venv
+public/dist
```

Comment:
This conflicts with the additions of `public/dist/app.min.[js|css]` below. If we're ever going to check those in, we should keep them in git history.